token-lending/cli/src/main.rs

```diff
@@ -0,0 +1,700 @@
+                )
+                .arg(
+                    Arg::with_name("loan_to_value_ratio")
+                        .long("loan-to-value-ratio")
+                        .validator(is_parsable::<u8>)
+                        .value_name("PERCENT")
+                        .takes_value(true)
+                        .required(true)
+                        .default_value("50")
+                        .help("Target ratio of the value of borrows to deposits: [0, 100)"),
+                )
+                .arg(
+                    Arg::with_name("liquidation_bonus")
+                        .long("liquidation-bonus")
+                        .validator(is_parsable::<u8>)
+                        .value_name("PERCENT")
+                        .takes_value(true)
+                        .required(true)
+                        .default_value("5")
+                        .help("Bonus a liquidator gets when repaying part of an unhealthy obligation: [0, 100]"),
+                )
+                .arg(
+                    Arg::with_name("liquidation_threshold")
+                        .long("liquidation-threshold")
+                        .validator(is_parsable::<u8>)
+                        .value_name("PERCENT")
+                        .takes_value(true)
+                        .required(true)
+                        .default_value("55")
+                        .help("Loan to value ratio at which an obligation can be liquidated: (LTV, 100]"),
+                )
+                .arg(
+                    Arg::with_name("min_borrow_rate")
+                        .long("min-borrow-rate")
+                        .validator(is_parsable::<u8>)
+                        .value_name("PERCENT")
+                        .takes_value(true)
+                        .required(true)
+                        .default_value("0")
+                        .help("Min borrow APY: min <= optimal <= max"),
+                )
+                .arg(
+                    Arg::with_name("optimal_borrow_rate")
+                        .long("optimal-borrow-rate")
+                        .validator(is_parsable::<u8>)
+                        .value_name("PERCENT")
+                        .takes_value(true)
+                        .required(true)
+                        .default_value("4")
+                        .help("Optimal (utilization) borrow APY: min <= optimal <= max"),
+                )
+                .arg(
+                    Arg::with_name("max_borrow_rate")
+                        .long("max-borrow-rate")
+                        .validator(is_parsable::<u8>)
+                        .value_name("PERCENT")
+                        .takes_value(true)
+                        .required(true)
+                        .default_value("30")
+                        .help("Max borrow APY: min <= optimal <= max"),
+                )
+                .arg(
+                    Arg::with_name("borrow_fee_wad")
```

Comment:
This isn't very user friendly. Can this be passed as a percent instead?

Reply:
Done.